develop/ide-tips-and-tricks.md

```diff
@@ -2,8 +2,9 @@
 title: IDE Tips and Tricks
 description: Useful information to handle and traverse your project using the IDE efficiently.
 authors:
-  - JR1811
-  - AnAwesomGuy
+  - anawesomguy
+  - imb11
+  - jr1811
```

Comment:
I didnt contribute this, this was OC from JR1811 from the original moddedmc.wiki